src/main/java/uk/gov/hmcts/reform/cwrdapi/servicebus/TopicPublisher.java

```diff
@@ -44,9 +44,13 @@ public void sendMessage(@NotNull Object message) {
                     ((PublishCaseWorkerData) message).getUserIds() != null
                             ? ((PublishCaseWorkerData) message).getUserIds().size() : null);
         }
-
-        jmsTemplate.convertAndSend(destination, message);
-        log.info("{}:: Message published to service bus topic", loggingComponentName);
+        try {
+            jmsTemplate.convertAndSend(destination, message);
+            log.info("{}:: Message published to service bus topic", loggingComponentName);
+        } catch (Exception ex) {
```

Comment:
Any reason for using try catch here? As we are retrying when the exception occurs, we need not throw every time. If all the attempts fail then we already have recover method to throw the exception

Reply:
Hi Sujeeth, I agree that the recover method will come into play when any JMSException occurs. This was a suggestion from Shashank as mentioned in ticket https://tools.hmcts.net/jira/browse/RDCC-2394 to put a try catch block in  sendMessage method. Let me cross verify with him and come back to you @shashank-rastogi  FYI.

Reply:
ah ok!! Please let me know after discussing with @shashank-rastogi . I shall approve the changes accordingly

Reply:
Hi @SujeethKumarAri As discussed with Shashank, we will rely on the Recover block for catching any exceptions. Removed the try/catch block.

Reply:
thanks @nitishsingla1234